### DISCUSSION TITLE
v-b-toggle, can I add additional arguements?

### DISCUSSION THREAD

**Discussion by DuffyZ** [bootstrap-vue/bootstrap-vue#5602] [Q&A]
Hi there,

This isn't a bug but a question.

When I toggle an accordion, I have to also set a value for :isDisabled: true or :isDisabled: false. When the accordion is open, the value should be true. When closed the value should be false.

I'm using :isDisabled to disable save buttons if an user has toggled the accordion open and is editing it.

Can I do this?

Here is my code:

```
<button href="#" v-b-toggle="'accordion-'+(index+1)" type="button" class="btn btn-sm btn-min-padding pull-right"><i class="fa fa-pencil" aria-hidden="true"></i></button>
```

**Comment by Hiws** [comment]
You can use the collapses `v-model` to control whether your button should be disabled or not.

https://codepen.io/Hiws/pen/eYJXZzN

**Comment by DuffyZ** [comment]
Your example doesn't help in my instance.

My code is rather complex. I create dynamic topics that are cards. 

The buttons I have are outside of the card. If I click on the toggle button on any of the cards, it has to disable these buttons.

Is there a class that v-b-toggle adds or removes?

I may have to create my own toggle function.

**Comment by Hiws** [comment]
> Is there a class that v-b-toggle adds or removes?

Yes, there is.
`v-b-toggle` adds the `collapsed` class if closed, and `not-collapsed` if opened.
